### PR TITLE
OCC-203: Update Guide

### PR DESCRIPTION
**Pull request by sarahelsaig**
[OCC-203](https://lombiq.atlassian.net/browse/OCC-203)

### CODE REVIEW

File: docs/guides/create-webshop.md
Comment (Piedone) [inline docs/guides/create-webshop.md:1]: The formatting is off: https://commerce.orchardcore.net/en/latest/features/exactly-payment/ See https://docs.orchardcore.net/en/latest/docs/topics/docs-contributions/ for running the docs site locally.

Comment (Piedone) [inline docs/guides/create-webshop.md:1]: Link src/Modules/OrchardCore.Commerce.Payment.Exactly/Readme.md from under "Payment providers" [here](https://github.com/OrchardCMS/OrchardCore.Commerce?tab=readme-ov-file#documentation) and also [here](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/main/docs/features/payment-providers.md).

File: docs/guides/create-webshop.md [hunk 1]
```diff
@@ -106,17 +106,17 @@ _Any content type with the Widget stereotype can be added this way, but let's ju
...
-_If you don't already have the necessary API keys, follow the links on the page._
+_See the links on the settings page and the [feature documentation](../features/exactly-payment.md) for info if you haven't got your keys yet._
 
-With that done, the Stripe Payment form now appears on the checkout page and the purchase can be completed.
+Once you're saved the settings, feel free to click on the _Verify currently saved API configuration_ to test your API access. With that done, the _Pay with exactly_ button now appears on the checkout page which redirects the customer to the payment processor's site and then if everything went well sends them back to the success page.
 
-![Stripe Payment.](../assets/images/create-webshop/step-6/stripe-form.png)
+![Exactly Payment.](../assets/images/create-webshop/step-6/exactly-form.png)
 
 _This will allow customers to pay to their card's content._
```
Comment (Skrypt) [inline docs/guides/create-webshop.md:121]:
```suggestion
_This will allow customers to pay to their cart content._
```

Comment (sarahelsaig) [reply docs/guides/create-webshop.md:121]: I believe that was a pun.

File: docs/guides/create-webshop.md [hunk 2]
```diff
@@ -106,17 +106,17 @@ _Any content type with the Widget stereotype can be added this way, but let's ju
 
 _Name the Widget and select the Layer where it should be rendered._
 
-### Step 6 — Enable a payment provider (Stripe)
+### Step 6 — Enable a payment provider (Exactly)
```
[the remark this answers is not in the document]
Comment (sarahelsaig) [reply docs/guides/create-webshop.md:109]: This guide describes a default setup and it's not a comprehensive walkthrough of all features. So I wouldn't keep the whole Stripe section. Rather we could add a line at the start of Step 6, like `If Exactly is not available in your region [try Stripe instead](../features/exactly-payment.md), the other payment provider OCC has built-in integration for.`

File: docs/guides/create-webshop.md (hunk 1 above)
Comment (Skrypt) [reply docs/guides/create-webshop.md:121]: Okay, I just got confused as a french canadian reader then. 😄

File: docs/guides/create-webshop.md
[the remark this answers is not in the document]
Comment (sarahelsaig) [reply docs/guides/create-webshop.md:1]: Ok I've added a link to the readme. The other links in the Documentation section just point to some of the same markdown files used when generating the docs site. Maybe it's a bit redundant, though useful if you check out past tags/commits. I don't think we have separate developer documentation (besides the xmldoc inside the source files).

File: docs/guides/create-webshop.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Piedone) [reply docs/guides/create-webshop.md:109]: That's OK.

File: docs/features/exactly-payment.md
Comment (Piedone) [inline docs/features/exactly-payment.md:1]: Exactly asks us to change the display name everywhere to include (R), i.e. "Exactly®".

Comment (sarahelsaig) [reply docs/features/exactly-payment.md:1]: Literally everywhere or just everywhere user facing?

Comment (Piedone) [reply docs/features/exactly-payment.md:1]: Just the display names, so user-facing only.

File: docs/features/exactly-payment.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
-# Exactly Payment
+# Exactly® Payment
 
-Orchard Core Commerce supports multiple [payment providers](payment-providers.md). [Exactly](https://exactly.com/) is one of the officially included ones. Unlike [our Stripe implementation](stripe-payment.md), it uses redirects to send you to a payment interface on their domain.
+Orchard Core Commerce supports multiple [payment providers](payment-providers.md). [Exactly®](https://exactly.com/) is one of the officially included ones. Unlike [our Stripe implementation](stripe-payment.md), it uses redirects to send you to a payment interface on their domain.
 
 To start using, follow these steps:
+
 1. Sign up to Exactly. Please be sure to use [this link](https://application.exactly.com/?utm_source=partner&utm_medium=kirill&utm_campaign=LOMBIQ) to create your account. That way [Lombiq](https://lombiq.com) (the steward of the Orchard Core Commerce project) will get a commission on the payment fees, which helps cover some of the development costs of OCC. This is at no cost to you; the fees you pay are the same either way.
```
Comment (DemeSzabolcs) [inline docs/features/exactly-payment.md:7]:
```suggestion
1. Sign up to Exactly®. Please be sure to use [this link](https://application.exactly.com/?utm_source=partner&utm_medium=kirill&utm_campaign=LOMBIQ) to create your account. That way [Lombiq](https://lombiq.com) (the steward of the Orchard Core Commerce project) will get a commission on the payment fees, which helps cover some of the development costs of OCC. This is at no cost to you; the fees you pay are the same either way.
```

File: docs/guides/create-webshop.md [hunk 4]
```diff
@@ -21,11 +21,11 @@ Get the latest version of the [Orchard Core Commerce NuGet](https://www.nuget.or
 
 ### Step 2 — Enable necessary features
 
-Certain Orchard Core Commerce features, as well as some prerequisite stock Orchard Core features, need to be enabled for the necessary functionality. This can be done under _Configuration > Features_. These features include the following:
+Certain Orchard Core Commerce features, as well as some prerequisite stock Orchard Core features, need to be enabled for the necessary functionality. This can be done under _Configuration > Features_, but if you used the _Orchard Core Commerce - Development_ setup recipe then they are already enabled. These features include the following:
 
 - **Orchard Core Commerce - Core**: Registers core components of Commerce features.
 - **Orchard Core Commerce - Payment** Provides the basics for online payment.
-- **Orchard Core Commerce - Payment - Stripe**: Provides online payment implementation using Stripe.
+- **Orchard Core Commerce - Payment - Exactly**: Provides online payment implementation using Exactly.
```
Comment (DemeSzabolcs) [inline docs/guides/create-webshop.md:28]:
```suggestion
- **Orchard Core Commerce - Payment - Exactly**: Provides online payment implementation using Exactly®.
```

File: docs/guides/create-webshop.md [hunk 5]
```diff
@@ -106,17 +106,19 @@ _Any content type with the Widget stereotype can be added this way, but let's ju
 
 _Name the Widget and select the Layer where it should be rendered._
 
-### Step 6 — Enable a payment provider (Stripe)
+### Step 6 — Enable a payment provider (Exactly®)
 
-Having Products and being able to browse them is great and all, but customers will also need a way to checkout and pay for their cart's content. This is where payment providers come into the picture. For simplicity's sake, we'll use Stripe as the payment provider here. Ensure the **Orchard Core Commerce - Payment - Stripe** feature is enabled, then go to _Configuration > Commerce > Stripe API_. The Publishable Key and Secret Key fields need to be filled in for the Stripe Payment form to work on the checkout page, see the links on the settings page for more.
+> ℹ If Exactly is not available in your region [try Stripe instead](../features/exactly-payment.md), the other payment provider OCC has built-in integration for.
```
Comment (DemeSzabolcs) [inline docs/guides/create-webshop.md:111]:
```suggestion
> ℹ If Exactly® is not available in your region [try Stripe instead](../features/exactly-payment.md), the other payment provider OCC has built-in integration for.
```

Comment (DemeSzabolcs) [reply docs/guides/create-webshop.md:111]: Also try Stripe links to the exactly-payment.md

File: src/Modules/OrchardCore.Commerce.Payment.Exactly/Views/CheckoutExactly.cshtml [hunk 6]
```diff
@@ -9,7 +9,7 @@
         T["An error has occurred while trying to connect to the payment service. Please try again later."];
 }
 
-<shape type="PayButton" PayButtonClass="exactly" prop-Text="@T["Pay with exactly"]" />
+<shape type="PayButton" PayButtonClass="exactly" prop-Text="@T["Pay with exactly\u00ae"]" />
```
Comment (DemeSzabolcs) [inline src/Modules/OrchardCore.Commerce.Payment.Exactly/Views/CheckoutExactly.cshtml:12]:
```suggestion
<shape type="PayButton" PayButtonClass="exactly" prop-Text="@T["Pay with Exactly\u00ae"]" />
```

**Review by DemeSzabolcs — CHANGES_REQUESTED** [review]

File: docs/guides/create-webshop.md (hunk 4 above)
Comment (sarahelsaig) [reply docs/guides/create-webshop.md:28]: That's the feature name, where I'd prefer to not use special characters.

**Review by DemeSzabolcs — APPROVED** [review]

**Pull request merged**